website/src/components/Messages/MessageTable.tsx

```diff
@@ -0,0 +1,27 @@
+import { Box, CircularProgress, Stack, StackDivider, useColorModeValue } from "@chakra-ui/react";
+import { MessageTableEntry } from "./MessageTableEntry";
+
+export function MessageTable({ messages, isLoading }) {
+  const backgroundColor = useColorModeValue("white", "gray.700");
+  const accentColor = useColorModeValue("gray.200", "gray.900");
+
+  return (
+    <Box
+      backgroundColor={backgroundColor}
+      boxShadow="base"
+      dropShadow={accentColor}
+      borderRadius="xl"
+      className="p-6 shadow-sm"
+    >
+      {isLoading ? (
```

Comment:
could you please move the `isLoading` check to the parent component?

Reply:
https://user-images.githubusercontent.com/19796430/210630496-7b82c897-2f84-468c-be6c-7e9f2f2f302f.mov

sure, i just had it in there to show small spinners... but could also change it to show a loading screen

Reply:
now removed... but still looks the same :)